src/main/java/duckie/Duckie.java

```diff
@@ -0,0 +1,70 @@
+package duckie;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.Scanner;
+
+import duckie.command.Command;
+import duckie.exception.DuckieException;
+import duckie.task.TaskList;
+
+/**
+ * Main file for the chatbot Duckie
+ */
+public class Duckie {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Instantiate the Duckie object together with the filePath of duckie file
+     * @param filePath Duckie file location
```

Comment:
Should there be a fullstop behind each param line because "Punctuation behind each parameter description"